vcd/resource_vcd_edgegateway.go

```diff
@@ -35,30 +35,13 @@ func resourceVcdEdgeGateway() *schema.Resource {
 				Optional: true,
 				ForceNew: true,
 			},
-			/*
-					// RFC:
-				    // Waiting on a decision whether such parameters are needed
-					"delete_force": &schema.Schema{
-						Type:        schema.TypeBool,
-						Required:    true,
-						ForceNew:    true,
-						Description: "TBD: no description in the API, and undefined behavior in practice",
-					},
-					"delete_recursive": &schema.Schema{
-						Type:        schema.TypeBool,
-						Required:    true,
-						ForceNew:    true,
-						Description: "TBD: no description in the API, and undefined behavior in practice",
-					},
-			*/
-			"advanced_networking": &schema.Schema{
+			"advanced": &schema.Schema{
 				Type:        schema.TypeBool,
-				Optional:    true,
-				Default:     false,
+				Required:    true,
 				ForceNew:    true,
-				Description: "True if the gateway uses advanced networking",
+				Description: "True if the gateway uses advanced networking. (Set by default in vCD 9.7+)",
```

Comment:
> Set by default in vCD 9.7+

This statement conflicts with `Required:    true`...

Reply:
adjusted. Now it is optional, with default `true`